github/tests/Repository.py

```diff
@@ -226,7 +226,7 @@ def testCreateIssueWithAllArgumentsStringLabel(self):
         self.assertEqual(issue.number, 30)
 
     def testCreateLabel(self):
-        label = self.repo.create_label("Label with silly name % * + created by PyGithub", "00ff00")
+        label = self.repo.create_label("Label with silly name % * + created by PyGithub", "00ff00", "Description of label with silly name")
         self.assertEqual(label.color, "00ff00")
         self.assertEqual(label.name, "Label with silly name % * + created by PyGithub")
         self.assertEqual(label.url, "https://api.github.com/repos/jacquev6/PyGithub/labels/Label+with+silly+name+%25+%2A+%2B+created+by+PyGithub")
```

Comment:
Perhaps you could add a assert description equal here?

Reply:
👍